src/main/java/com/box/sdk/BoxZipItem.java

```diff
@@ -0,0 +1,41 @@
+package com.box.sdk;
+
+/**
+ * Represents a Box item to be included when creating a zip file.
+ */
+public class BoxZipItem extends BoxJSONObject {
+    private String type;
+    private String id;
+
+    /**
+     * Constructs a base BoxZipItem object.
+     *
+     * @param type item type, "file" or "folder".
+     * @param id   id of the the item.
+     */
+    public BoxZipItem(String type, String id) {
+        super();
+        this.type = type;
+        this.id = id;
+        this.addPendingChange("type", type);
+        this.addPendingChange("id", id);
```

Comment:
?

Reply:
Let's remove `extends BoxJSONObject` and instead add a custom method to return a jsonObject.  Thanks!